src/apis/servicing_api.ts

```diff
@@ -149,6 +147,32 @@ export class ServicingAPI {
         );
     }
 
+    /**
+     * Asynchronously retrieve the `DebtRegistryEntry` instance mapped to the
+     * issuance hash specified.
+     *
+     * @param  issuanceHash the id of the issuance to retrieve.
+     * @return              the relevant `DebtRegistryEntry` instance .
+     */
+    public async getDebtRegistryEntry(issuanceHash: string): Promise<DebtRegistryEntry> {
+        this.assert.schema.bytes32("issuanceHash", issuanceHash);
+
+        let {
+            debtToken,
+            repaymentRouter,
+            debtRegistry,
+            tokenTransferProxy,
+        } = await this.contracts.loadDharmaContractsAsync();
```

Comment:
Please either only pull `debtToken` from `loadDharmaContractsAsync`, or just pull `debtToken` from `loadDebtTokenContractAsync`.

The performance difference is negligible so really it's up to you.